### PR TITLE
Sort accents

### PR DESCRIPTION
**Pull request by tobiasKaminsky**
ref: #1110

### CODE REVIEW

**Comment by Kernald** [comment]
While your fix to #1110 looks fine to me (and that's the really important part of this comment!), I have two remarks: your two first commits (6df6548 and 69e7869) are unrelated to this change, and you made a lot of space/newline changes. It makes reviewing more complicated, it would be great to avoid it (or at least, change it in a specific commit, maybe? I don't know the usage in this repository).

**Comment by tobiasKaminsky** [comment]
You are right.
But I doubt that this minor optical clean up would be merged if it is a single PR ;)

And I do not like the new lines before {, e.g:

if (test == 0) 
{ 
do this
}

is nasty and should be:
if (test == 0){
  do this
}

**Comment by Kernald** [comment]
I agree totally on the style ;-) If this happens again, maybe you should split the changes: a first commit to change the style, then the actual code change… As I said, I don't know if there's an usual way to work with these kinds of things at OwnCloud, it's just a personnal feedback :-)

**Comment by AndyScherzinger** [comment]
I also reviewed the code and it looks absolutely fine to, imho ready for QA then: @davivel @rperezb 

+1

**Comment by rperezb** [comment]
Thanks!

File: src/com/owncloud/android/ui/adapter/FileListListAdapter.java [hunk 1]
```diff
@@ -326,6 +326,8 @@ public View getView(int position, View convertView, ViewGroup parent) {
                         Drawable backrepeat = mContext.getResources().
                                               getDrawable(R.drawable.backrepeat);
                         fileIcon.setBackground(backrepeat);
+                    } else {
+                        fileIcon.setBackground(null);
```
Comment (masensio) [inline src/com/owncloud/android/ui/adapter/FileListListAdapter.java]: Is this line necessary for the fix?

I think this line is from another fix (https://github.com/owncloud/android/pull/1113 and https://github.com/owncloud/android/pull/1114). Besides at the end this line had been removed in this fix.

Maybe, if you update this PR with the last version of master branch this line will disappear.

File: src/com/owncloud/android/ui/adapter/FileListListAdapter.java [hunk 2]
```diff
@@ -335,6 +337,8 @@ public View getView(int position, View convertView, ViewGroup parent) {
 
             } else {
                 // Folder
+                fileIcon.setBackground(null);
```
Comment (masensio) [inline src/com/owncloud/android/ui/adapter/FileListListAdapter.java]: Same comment as in the previous line.

**Comment by tobiasKaminsky** [comment]
@masensio You are right. This is now replaced :)

**Comment by masensio** [comment]
Thanks @tobiasKaminsky for the changes.
So PR Ready to be tested :)

**Comment by rperezb** [comment]
@Dianafg76 can you please take care of QA this? 
thx!

**Comment by Dianafg76** [comment]
I tested and it is working OK
Smsung Galaxy S3 v 4.1.2

**Comment by masensio** [comment]
thanks @Dianafg76

**Pull request merged**